modules/margin-protocol/src/tests.rs

```diff
@@ -35,7 +30,7 @@ fn risk_threshold(margin_call_percent: u32, stop_out_percent: u32) -> RiskThresh
 	}
 }
 
-fn eur_jpy_long() -> Position<Runtime> {
+fn jpy_eur_long() -> Position<Runtime> {
```

Comment:
What's the reason of renaming?

Reply:
Keep the same as `pair: JPY_EUR_PAIR`, but it seems wrong. I will rename it.